### PR TITLE
Ds small corrections

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Minor formal revisions.

### CODE REVIEW

File: DayrSuryan/DSEthiop3.xml [hunk 1]
```diff
@@ -760,6 +753,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                               <locus from="101r" to="102v"/>
                            </extent>
                            <collation>
+                             In <ref target="#p1">Unit 1</ref>, the dimensions of the written area of and the number ow written lines slightly change
```
Comment (thea-m) [inline DayrSuryan/DSEthiop3.xml:756]: I think that this information would be better kept in layout, not collation

File: DayrSuryan/DSEthiop1.xml [hunk 2]
```diff
@@ -664,7 +664,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   <bindingDesc>
                      <binding xml:id="binding">
                         <decoNote xml:id="b1">
-                           Two wooden boards covered with reddish brown blind-tooled leather.
+                           Bound in Ethiopia. Two wooden boards covered with reddish brown blind-tooled leather.
```
Comment (thea-m) [inline DayrSuryan/DSEthiop1.xml:667]: is there a conceptual difference between "Bound in Ethiopia" here and "Ethiopian" elsewhere?

Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop1.xml:667]: It should be "Ethiopian binding".

**Comment by DenisNosnitsin1970** [comment]
S. DS Ethiop 3 - layout like this? we want that this info would be visible in the catalogue.

**Comment by thea-m** [comment]
it will not be visible yet, but I wrote it down, we will find a solution

**Comment by DenisNosnitsin1970** [comment]
Thanks!
We want to give the reader creadible information :)

**Pull request merged**